### ISSUE TITLE
List of key-value pairs

### ISSUE THREAD

**Issue by AlexITMan** [thunkable/thunkable-issues#198]
An index is used to access a list item, but in many tasks it is much more convenient to access by key name.

The list of pairs is easy to do in Thunkable X, but for beginners it can cause difficulties and the need to go a more complicated way using indexes.

**Comment by mark-friedman** [comment]
You can currently do something similar with variables, since you can get and set variables by a Text value, which you can think of as a key name.

That said, we are in the process of adding support for using Text values as Object property references, which, I believe, is essentially what you are asking for here.

**Comment by AlexITMan** [comment]
You are right, but it is much more convenient to use a list of pairs instead of a set of variables.

I think the "get property of object" block is better for this, since JSON allows to create a list of pairs. But for this need in this block to use a variable to specify the name of the field (I will add this query).

**Comment by mark-friedman** [comment]
As I said, we are currently in the process of adding that block.
